packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -794,7 +794,7 @@ class _TimePickerDialogState extends State<_TimePickerDialog> {
 ///  * <https://material.google.com/components/pickers.html#pickers-time-pickers>
 Future<TimeOfDay> showTimePicker({
   BuildContext context,
```

Comment:
It does not.  For example, the `onPressed` property of buttons are `@required` but allowed to be null.  That's because passing `null` has the side effect of disabling the button and we want developers to do that explicitly so they don't have a mysteriously disabled button by default.